agent_workflows/config.yaml

```diff
@@ -1,16 +1,23 @@
 api: "sambastudio"
 
-embedding_model: "sambastudio"
+embedding_model: 
+    "type": "cpu" # set either sambastudio or cpu
+    "batch_size": 1 #set depending of your endpoint configuration (1 if CoE embedding expert)
+    "coe": True #set true if using Sambastudio embeddings in a CoE endpoint 
+    "select_expert": "e5-mistral-7b-instruct" #set if using SambaStudio CoE embedding expert
 
 loaders: 
     pdf: "unstructured"
     txt: "text_loader"
 
 llm: 
     "temperature": 0.0
-    "max_tokens_to_generate": 1024
-    "sambaverse_model_name": "Meta/llama-3-70b-chat-hf"
-    "sambaverse_select_expert": "Meta-Llama-3-70B-Instruct"
+    "do_sample": False
+    "max_tokens_to_generate": 1200
+    "sambaverse_model_name": "Meta/Meta-Llama-3-8B-Instruct"
+    "coe": True #set as true if using Sambastudio CoE endpoint
+    "select_expert": "Meta-Llama-3-8B-Instruct" #set if using FastCoE, Sambaverse or SambaStudio CoE llm expert
+    #fastApi CoE expert name -> "llama3-8b"
```

Comment:
Any reason this is commented out @snova-luiss ? Thanks

Reply:
Not code, it’s just the continuation of the instruction from the previous line